service/worker/addsearchattributes/workflow.go

```diff
@@ -152,6 +155,20 @@ func (a *activities) AddESMappingFieldActivity(ctx context.Context, params Workf
 	return nil
 }
 
+func (a *activities) isRetryableError(err error) bool {
+	var httpStatusCode int
+	if err, isElasticErr := err.(*elastic.Error); isElasticErr {
+		httpStatusCode = err.Status
+	}
+
+	switch httpStatusCode {
+	case http.StatusBadRequest, http.StatusUnauthorized, http.StatusForbidden, http.StatusNotFound, http.StatusConflict:
```

Comment:
https://discuss.elastic.co/t/knowing-when-and-when-not-to-retry-a-request-based-on-elasticsearchexception-or-ioexception-with-the-resthighlevelclient/183779